x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/EmptyPrompt.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiEmptyPrompt } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { CreateCustomLinkButton } from './CreateCustomLinkButton';
+
+export const EmptyPrompt = ({
+  onCreateCustomLinkClick
+}: {
+  onCreateCustomLinkClick: () => void;
+}) => {
+  return (
+    <EuiEmptyPrompt
+      iconType="boxesHorizontal"
```

Comment:
```suggestion
      iconType="link"
```

Since we changed to links, I think this icon fits better.